pkg/kubelet/dockertools/manager.go

```diff
@@ -64,6 +64,8 @@ const (
 	// we want to able to consider SRV lookup names like _dns._udp.kube-dns.default.svc to be considered relative.
 	// hence, setting ndots to be 5.
 	ndotsDNSOption = "options ndots:5\n"
+
+	containerNetworkEnvKey = "NETWORK"
```

Comment:
Maybe like the kubernetes (and old kubernetes-ro) service we could have something statically defined in the api server. The kube-proxy could then just forward to the kubelet on localhost. Does mean you would need DNS to get the service IP (or use the ENV var, but I know the project as a whole seems to be steering away from ENV vars), to ask about your network configuration. Not sure if that's a problem in anyone's mind.